### PR TITLE
update notebooks for 5.0.3 and fix metrics plugin versions

### PR DESCRIPTION
**Pull request by pratapkishorevarma**

### CODE REVIEW

File: Cloud Pak for Data/Batch Support/4.5/hive/Watson OpenScale - Monitor models with data in Hive.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.5/hive/Watson OpenScale - Monitor models with data in Hive.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.5.0` to `!pip install "ibm_wos_utils~=4.5.0"`

File: Cloud Pak for Data/Batch Support/4.5/jdbc/Watson OpenScale - Monitor models with data in DB2.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.5/jdbc/Watson OpenScale - Monitor models with data in DB2.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.5.0` to `!pip install "ibm_wos_utils~=4.5.0"`

File: Cloud Pak for Data/Batch Support/4.6/hive/Watson OpenScale - Monitor models with data in Hive.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.6/hive/Watson OpenScale - Monitor models with data in Hive.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.5.0` to `!pip install "ibm_wos_utils~=4.6.0"`

File: Cloud Pak for Data/Batch Support/4.6/hive/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.6/hive/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.6.0` to `!pip install "ibm_wos_utils~=4.6.0"`

File: Cloud Pak for Data/Batch Support/4.6/hive/samples/Watson OpenScale and Batch - WML - IAE - hive.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.6/hive/samples/Watson OpenScale and Batch - WML - IAE - hive.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.6.0` to `!pip install "ibm_wos_utils~=4.6.0"`

File: Cloud Pak for Data/Batch Support/4.6/jdbc/Watson OpenScale - Monitor models with data in DB2.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.6/jdbc/Watson OpenScale - Monitor models with data in DB2.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.5.0` to `!pip install "ibm_wos_utils~=4.6.0"`

File: Cloud Pak for Data/Batch Support/4.6/jdbc/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.6/jdbc/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.6.0` to `!pip install "ibm_wos_utils~=4.6.0"`

File: Cloud Pak for Data/Batch Support/4.6/jdbc/samples/Watson OpenScale and Batch - WML - IAE - JDBC.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.6/jdbc/samples/Watson OpenScale and Batch - WML - IAE - JDBC.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.6.0` to `!pip install "ibm_wos_utils~=4.6.0"`

File: Cloud Pak for Data/Batch Support/4.7/hive/Watson OpenScale - Monitor models with data in Hive.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.7/hive/Watson OpenScale - Monitor models with data in Hive.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.7.0` to `!pip install "ibm_wos_utils~=4.7.0"`

File: Cloud Pak for Data/Batch Support/4.7/hive/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.7/hive/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.7.0` to `!pip install "ibm_wos_utils~=4.7.0"`

File: Cloud Pak for Data/Batch Support/4.7/hive/samples/Watson OpenScale and Batch - WML - IAE - hive.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.7/hive/samples/Watson OpenScale and Batch - WML - IAE - hive.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.7.0` to `!pip install "ibm_wos_utils~=4.7.0"`

File: Cloud Pak for Data/Batch Support/4.7/jdbc/Watson OpenScale - Monitor models with data in DB2.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.7/jdbc/Watson OpenScale - Monitor models with data in DB2.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.7.0` to `!pip install "ibm_wos_utils~=4.7.0"`

File: Cloud Pak for Data/Batch Support/4.7/jdbc/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.7/jdbc/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.7.0` to `!pip install "ibm_wos_utils~=4.7.0"`v

File: Cloud Pak for Data/Batch Support/4.7/jdbc/samples/Watson OpenScale and Batch - WML - IAE - JDBC.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.7/jdbc/samples/Watson OpenScale and Batch - WML - IAE - JDBC.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.7.0` to `!pip install "ibm_wos_utils~=4.7.0"`

File: Cloud Pak for Data/Batch Support/4.8/hive/Watson OpenScale - Onboard models for monitoring using Configuration Package.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.8/hive/Watson OpenScale - Onboard models for monitoring using Configuration Package.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.8.0` to `!pip install "ibm_wos_utils~=4.8.0"`

File: Cloud Pak for Data/Batch Support/4.8/hive/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.8/hive/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.8.0` to `!pip install "ibm_wos_utils~=4.8.0"`

File: Cloud Pak for Data/Batch Support/4.8/hive/samples/Watson OpenScale and Batch - WML - IAE - hive.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.8/hive/samples/Watson OpenScale and Batch - WML - IAE - hive.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.8.0` to `!pip install "ibm_wos_utils~=4.8.0"`

File: Cloud Pak for Data/Batch Support/4.8/jdbc/Watson OpenScale - Onboard models for monitoring using Configuration Package.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.8/jdbc/Watson OpenScale - Onboard models for monitoring using Configuration Package.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.8.0` to `!pip install "ibm_wos_utils~=4.8.0"`

File: Cloud Pak for Data/Batch Support/4.8/jdbc/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.8/jdbc/Watson OpenScale - Onboard models for monitoring using training data table and a sample csv.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.8.0` to `!pip install "ibm_wos_utils~=4.8.0"`

File: Cloud Pak for Data/Batch Support/4.8/jdbc/samples/Watson OpenScale and Batch - WML - IAE - JDBC.ipynb
Comment (prempiyush) [inline Cloud Pak for Data/Batch Support/4.8/jdbc/samples/Watson OpenScale and Batch - WML - IAE - JDBC.ipynb:1]: Please change this `!pip install ibm_wos_utils>=4.8.0` to `!pip install "ibm_wos_utils~=4.8.0"`

File: IBM Cloud/AWS Sagemaker/notebooks/Credit  model with SageMaker linear-learner .ipynb
Comment (prempiyush) [inline IBM Cloud/AWS Sagemaker/notebooks/Credit  model with SageMaker linear-learner .ipynb:1]: If sagemaker is not being used, perhaps better to delete the installation line.

File: WatsonX.Governance/Cloud/GenAI/samples/Detached Prompt Evaluation HF FLAN model.ipynb
Comment (prempiyush) [inline WatsonX.Governance/Cloud/GenAI/samples/Detached Prompt Evaluation HF FLAN model.ipynb:1]: No command to install ibm_watsonx_ai?

File: WatsonX.Governance/Cloud/GenAI/samples/Detached Prompt Evaluation RAG task type.ipynb
Comment (prempiyush) [inline WatsonX.Governance/Cloud/GenAI/samples/Detached Prompt Evaluation RAG task type.ipynb:1]: No command to install ibm_watsonx_ai?

**Review by prempiyush — CHANGES_REQUESTED** [review]
General comment. 

For all the files changed, I think we should change the copyright year too.

**Comment by ravichamarthy** [comment]
When you changes all the files with the metrics plugin version, did you also run all the notebooks to make sure they are working fine?

**Review by sokollip — APPROVED** [review]

**Comment by pratapkishorevarma** [comment]
> General comment.
> 
> For all the files changed, I think we should change the copyright year too.

Only few notebooks have copyright footer. If we want to add them, need to handle as part of a new task.

**Review by prempiyush — APPROVED** [review]

**Comment by csgorham** [comment]
keep getting this error "numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject"  tried ```!pip install  pybind11>=2.12
!pip install numpy==1.26.4'''

**Pull request merged**